tasks/lammps/native.py

```diff
@@ -0,0 +1,41 @@
+from invoke import task
```

Comment:
This is showing up as new in my diff, but it's actually just been moved from the `tasks` dir into `tasks/lammps`.